cmake/FindDummy.cmake

```diff
@@ -0,0 +1,102 @@
+#####################
+## FindDummy.cmake
```

Comment:
Maybe rename to .cmake.template just to make sure that this is not a cmake file that can be included.

Reply:
:+1: